homeassistant/components/conversation/default_agent.py

```diff
@@ -592,6 +610,80 @@ def _get_error_text(
         response_str = lang_intents.error_responses.get(response_key)
         return response_str or _DEFAULT_ERROR_TEXT
 
+    def register_trigger(
+        self,
+        sentences: list[str],
+        callback: core.CALLBACK_TYPE,
+        response: str | None = None,
```

Comment:
Let's have the callback return the response. That way the user could define different responses inside their script in the future.